intro-HPC/ch_troubleshooting.tex

```diff
@@ -91,6 +91,20 @@ \section{Issues connecting to login node}
             share your private key, they're called \emph{private} for a good reason.
 \end{enumerate}
 
+\ifwindows
+
+If you're using PuTTY and get this errormessage:
+
+\begin{prompt}
+server unexpectedly closed network connection
+\end{prompt}
+
+it can be because your PuTTY version is too old. Make sure you're using at least
```

Comment:
`it is possible that the PuTTY version you are using is too old and doesn't support some required (security-related) features.`

`you're` -> `you are`